docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -4,75 +4,370 @@ title: "Red Hat OpenShift"
 
+If you need to set up an OpenShift cluster on a cloud provider, we recommend our [guide to deploying a ROSA cluster](/self-managed/setup/deploy/amazon/openshift/terraform-setup.md).
+
 ### Supported Versions
 
 We conduct testing and ensure compatibility against the following OpenShift versions:
 
 | OpenShift Version | [End of Support Date](https://access.redhat.com/support/policy/updates/openshift) |
 | ----------------- | --------------------------------------------------------------------------------- |
+| 4.17.x            | June 27, 2025                                                                     |
 | 4.16.x            | December 27, 2025                                                                 |
 | 4.15.x            | August 27, 2025                                                                   |
 | 4.14.x            | May 1, 2025                                                                       |
-| 4.13.x            | November 17, 2024                                                                 |
 
-:::caution
+:::caution Older versions are not guaranteed
+
 Compatibility is not guaranteed for OpenShift versions no longer supported by Red Hat, as per the End of Support Date. For more information, refer to the [Red Hat OpenShift Container Platform Life Cycle Policy](https://access.redhat.com/support/policy/updates/openshift).
+
 :::
 
-## Deploying Camunda 8 in OpenShift
+## Requirements
 
-Depending on your OpenShift cluster's Security Context Constraints (SCCs) configuration, the deployment process may vary.
+- [Helm (3.16+)](https://helm.sh/docs/intro/install/)
+- [kubectl (1.30+)](https://kubernetes.io/docs/tasks/tools/#kubectl) to interact with the cluster.
+- [jq (1.7+)](https://jqlang.github.io/jq/download/) to interact with some variables.
+- [GNU envsubst](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html) to generate manifests.
+- [oc (version supported by your OpenShift)](https://docs.openshift.com/container-platform/4.17/cli_reference/openshift_cli/getting-started-cli.html) to interact with OpenShift.
+- A namespace to host the Camunda Platform, in this guide we will reference `camunda` as the target namespace.
 
-<Tabs queryString="current-scc">
-<TabItem value="w-scc" label="Security Context Constraints (SCCs)" default>
+## Deploy Camunda 8 via Helm charts
+
+### Configure your deployment
+
+Start by creating a `values.yml` file to store the configuration for your environment.
+This file will contain key-value pairs that will be substituted using `envsubst`.
+Over this guide, you will add and merge values in this file to configure your deployment to fit your needs.
+
+You can find a reference example of this file here:
+
+```hcl reference
+https://github.com/camunda/camunda-deployment-references/blob/feature/openshift-ra-standard/aws/rosa-hcp/camunda-versions/8.7/procedure/install/helm-values/base.yml
```

Comment:
would it make sense to call this file `values.yml`?

I called mine `base.yml` as the file was called that way.